src/js/rise-cache.js

```diff
@@ -44,10 +45,21 @@ RiseVision.Common.RiseCache = (function () {
             // call ping again so correct ping URL is used for Rise Cache V2
             return self.ping(callback);
           } else {
-            console.debug("Rise Cache is not running");
-            _isCacheRunning = false;
 
-            callback(false, null);
+            if ( _isHttp ) {
```

Comment:
@rodrigopavezi What is the status of a failed `http://localhost:9494` ping? If it's 404, this code will not execute and instead will be in an infinite loop of executing the block above.